src/main/java/ch/njol/skript/expressions/ExprSpectatorTarget.java

```diff
@@ -24,60 +24,151 @@
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
-import ch.njol.skript.classes.Changer;
-import ch.njol.skript.doc.NoDoc;
-import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import com.destroystokyo.paper.event.player.PlayerStartSpectatingEntityEvent;
+import com.destroystokyo.paper.event.player.PlayerStopSpectatingEntityEvent;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.data.DefaultChangers;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.effects.Delay;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-@NoDoc
-public class ExprSpectatorTarget extends SimplePropertyExpression<Player, Entity> {
+@Name("Spectator Target")
+@Description("Grabs the spectator target entity of the players.")
+@Examples({
+	"on player start spectating of player:",
+		"\tmessage \"&c%spectator target% currently has %{game::kills::%spectator target%}% kills!\" to the player",
+	"",
+	"on player stop spectating:",
+		"\tpast spectator target was a zombie",
+		"\tset spectator target to the nearest skeleton"
+})
+@RequiredPlugins("Paper")
```

Comment:
I would specify *when* Paper is required

Reply:
Spectating has been out since 1.9 Skript supports 1.13+